fetch.bs

```diff
@@ -5329,11 +5363,11 @@ method, must run these steps:
 <ol>
  <li><p>Let <var>p</var> be a new promise.
 
- <li><p>Let <var>request</var> be the associated
- <a for=/>request</a> of the result of invoking the initial value of
- {{Request}} as constructor with <var>input</var> and
- <var>init</var> as arguments. If this throws an exception, reject
- <var>p</var> with it and return <var>p</var>.
+ <li><p>Let <var>requestObject</var> be the result of invoking the initial value of {{Request}} as
+ constructor with <var>input</var> and <var>init</var> as arguments. If this throws an exception,
+ reject <var>p</var> with it and return <var>p</var>
+
+ <li><p>Let <var>request</var> be <var>requestObject</var>'s <a for=Request>request</a>.
```

Comment:
It introduces *requestObject* as a variable, so I can get its signal in the proceeding steps.